### ISSUE TITLE
[BUG]: property-no-unknown feature is not compatible with other feature

### ISSUE THREAD

**Issue by rehberbey** [stylelint-scss/stylelint-scss#667]
```scss
.foo {
    text-transform: uppercase; // ❌
    text-decoration: none; // ❌
 /* ^? Expected property "text-transform" to be in a nested form
    (scss/declaration-nested-properties) Stylelintscss/declaration-nested-properties */
}
```

Feature **scss/declaration-nested-properties** asks us to be like this.
But the **property-no-unknown** property does not allow it:

```scss
.foo {
    text {
        transform: uppercase;
        decoration: none; // ❌
    /*  ^? Unexpected unknown property "decoration"
        (property-no-unknown) Stylelintproperty-no-unknown */
    }
}
```

**Comment by pamelalozano16** [comment]
You are missing a colon after `text`, so it's not parsing correctly as a nested property.
```scss
.foo { 
  text: {
    transform: uppercase;
    decoration: none;
  }
}
```
